src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -254,5 +269,20 @@ private static string GetMethodDocumentation(IMethodSymbol method, Compilation c
                     return method.GetDocumentationComment(compilation, expandIncludes: true, expandInheritdoc: true, cancellationToken: cancellationToken).SummaryText;
             }
         }
+
+        private static string GetMethodReturnsDocumentation(IMethodSymbol method, Compilation compilation, CancellationToken cancellationToken)
```

Comment:
Inline this into the other method somehow? Or just write GetReturnsDocumentation to be something along the lines of:

```
if (symbol.IsPropertyAccessor() || ...) { symbol = symbol.AssociatedSymbol; } 

return symbol.GetDocumentationComment(compilation, expandIncludes: true, expandInheritdoc: true, cancellationToken: cancellationToken).ReturnsText
```

As it stands that "fetch the documentation comment itself and do .ReturnsText" exists in now three places in this file; any edit to one probably requires editing the other two.

Reply:
I'll make sure this is a bit simpler in the next PR.